bin/dbconsole

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env python3
+
+import os
+import tomllib
+from pathlib import Path
+
+env = os.getenv('APP_ENV', 'development')
+config = tomllib.loads(Path('config/database.toml').read_text())[env]
+
+os.execv(
+    '/usr/bin/mysql', [
+        '/usr/bin/mysql',
+        '--local-infile',
+        f'-u{config["username"]}',
+        f'-p{config["password"]}',
+        config['database']
+    ]
+)
```

Comment:
This is not necessary for tests, but it's pretty convenient. For instance, loading a database dump with `bin/dbconsole < prod_dump.sql`.